x-pack/plugins/apm/public/components/shared/span_links/span_links_tab_content.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiNotificationBadge, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { SpanLinks } from '.';
+import { SpanLinksCount } from '../../app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/waterfall_helpers/waterfall_helpers';
+import { ProcessorEvent } from '../../../../common/processor_event';
+
+interface Props {
+  spanLinksCount: SpanLinksCount;
+  traceId: string;
+  spanId: string;
+  processorEvent: ProcessorEvent;
+}
+
+export function getSpanLinksTabContent({
+  spanLinksCount,
+  traceId,
+  spanId,
+  processorEvent,
+}: Props) {
+  if (!spanLinksCount.linkedChildren && !spanLinksCount.linkedParents) {
+    return;
```

Comment:
can you explicitly return undefined here?

Reply:
because it's more explicit. we're returning a value in other cases. It makes it easier to understand what the function is doing. I thought we had a lint rule for this, but apparently not.